cmd/controller/ctrl.go

```diff
@@ -59,13 +59,7 @@ func NewController(
 
 // getUid will return unique id for the disk block device
 func getUid(blk v1.Blockdevice) string {
```

Comment:
Make it as exported function. This will use for disk label in iostat metrics.

Reply:
lets keep that change as a part of iostat metrics commit only, as this function is not needed to be exported here in this change.

Reply:
Ok.